nx-X11/programs/Xserver/hw/nxagent/X11/include/Xrender_nxagent.h

```diff
@@ -16,7 +16,7 @@
  * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS, IN NO EVENT SHALL SuSE
```

Comment:
HI Mihai,

On  Do 21 Apr 2016 03:25:39 CEST, Mihai Moldovan wrote:

>> @@ -16,7 +16,7 @@
>>   * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS, IN NO EVENT  
>> SHALL SuSE
>
> Do we really still need this file? It's a copy of `libXrender`'s  
> `Xrender.h` file, which should normally be directly includable.

Unfortunately, we _do_ need this file, because we cannot build against  
libnx-x11 and libx11 at the same time. Meaning, we cannot include  
<X11/Xlib.h> and <nx-X11/Xlib.h> as types and structures of various  
major components have changed (e.g. GCstruct, Display, etc.).

To work around this, we ship the fake header files of the libXrender  
(and other libX* libraries) that include Xlib.h from nx-X11.

Mike

-- 

DAS-NETZWERKTEAM
mike gabriel, herweg 7, 24357 fleckeby
mobile: +49 (1520) 1976 148
landline: +49 (4354) 8390 139

GnuPG Key ID 0x25771B31
mail: mike.gabriel@das-netzwerkteam.de, http://das-netzwerkteam.de

freeBusy:
https://mail.das-netzwerkteam.de/mailxchange/kronolith/fb.php?u=m.gabriel%40das-netzwerkteam.de